{"1.4.1":{"info":{"author":"Anders Krogh Mortensen","author_email":"anderskroghm@gmail.com","bugtrack_url":null,"classifiers":["Programming Language :: Python :: 3","Programming Language :: Python :: 3.12"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/atarwn/GPUtil-fix","keywords":null,"license":null,"maintainer":"atarwn","maintainer_email":"a@qwa.su","name":"GPUtil-fix","package_url":"https://pypi.org/project/GPUtil-fix/","platform":null,"project_url":"https://pypi.org/project/GPUtil-fix/","project_urls":{"Homepage":"https://github.com/atarwn/GPUtil-fix"},"provides_extra":null,"release_url":"https://pypi.org/project/GPUtil-fix/1.4.1/","requires_dist":null,"requires_python":null,"summary":"Utility to get the GPU status from NVIDA GPUs using nvidia-smi","version":"1.4.1","yanked":true,"yanked_reason":"In this release, I forgot to change `spawn.find_executable` to `shutil.which`, so you would get an error when using this module"},"last_serial":24254433,"urls":[{"comment_text":"","digests":{"blake2b_256":"9a13075c9737d69de033c376d30c3fbadf72d144230b768804a4de45e1593033","md5":"c3f593f52ba30604af006ee7937d1b9e","sha256":"7be6b5e3c82b293600e7ba10a82fff88f0b3286aa7d3d3571eea3e43efb3c44a"},"downloads":-1,"filename":"GPUtil_fix-1.4.1-py3-none-any.whl","has_sig":false,"md5_digest":"c3f593f52ba30604af006ee7937d1b9e","packagetype":"bdist_wheel","python_version":"py3","requires_python":null,"size":13891,"upload_time":"2024-07-23T12:11:41","upload_time_iso_8601":"2024-07-23T12:11:41.249395Z","url":"https://files.pythonhosted.org/packages/9a/13/075c9737d69de033c376d30c3fbadf72d144230b768804a4de45e1593033/GPUtil_fix-1.4.1-py3-none-any.whl","yanked":true,"yanked_reason":"In this release, I forgot to change `spawn.find_executable` to `shutil.which`, so you would get an error when using this module"},{"comment_text":"","digests":{"blake2b_256":"92de575848ed6e3dd62ec0b2377447edc793ece5aeb533de1909439127d70e65","md5":"50d1eedb37de9ffdbad817fecae61961","sha256":"08ece8d7e60e862e3af90367c413c5341f214e575b7522d1203b48acab600bb9"},"downloads":-1,"filename":"gputil_fix-1.4.1.tar.gz","has_sig":false,"md5_digest":"50d1eedb37de9ffdbad817fecae61961","packagetype":"sdist","python_version":"source","requires_python":null,"size":18453,"upload_time":"2024-07-23T12:11:43","upload_time_iso_8601":"2024-07-23T12:11:43.158182Z","url":"https://files.pythonhosted.org/packages/92/de/575848ed6e3dd62ec0b2377447edc793ece5aeb533de1909439127d70e65/gputil_fix-1.4.1.tar.gz","yanked":true,"yanked_reason":"In this release, I forgot to change `spawn.find_executable` to `shutil.which`, so you would get an error when using this module"}],"vulnerabilities":[]},"1.4.1.2":{"info":{"author":"Anders Krogh Mortensen","author_email":"anderskroghm@gmail.com","bugtrack_url":null,"classifiers":["Programming Language :: Python :: 3","Programming Language :: Python :: 3.12"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/atarwn/GPUtil-fix","keywords":null,"license":null,"maintainer":"atarwn","maintainer_email":"a@qwa.su","name":"GPUtil-fix","package_url":"https://pypi.org/project/GPUtil-fix/","platform":null,"project_url":"https://pypi.org/project/GPUtil-fix/","project_urls":{"Homepage":"https://github.com/atarwn/GPUtil-fix"},"provides_extra":null,"release_url":"https://pypi.org/project/GPUtil-fix/1.4.1.2/","requires_dist":null,"requires_python":null,"summary":"Utility to get the GPU status from NVIDA GPUs using nvidia-smi","version":"1.4.1.2","yanked":false,"yanked_reason":null},"last_serial":24254433,"urls":[{"comment_text":"","digests":{"blake2b_256":"c980b5fc15416c4f4d2e12d9c7a99f26a104465ada4eb2e7de0107eb71aa064f","md5":"aef323878f1e4f5ddf289620f97a98f0","sha256":"ab0ff31b9312c4f161d8e97705f08d7f36eb6fc00f150232ac8145256d80d2be"},"downloads":-1,"filename":"GPUtil_fix-1.4.1.2-py3-none-any.whl","has_sig":false,"md5_digest":"aef323878f1e4f5ddf289620f97a98f0","packagetype":"bdist_wheel","python_version":"py3","requires_python":null,"size":13945,"upload_time":"2024-07-23T12:51:05","upload_time_iso_8601":"2024-07-23T12:51:05.194449Z","url":"https://files.pythonhosted.org/packages/c9/80/b5fc15416c4f4d2e12d9c7a99f26a104465ada4eb2e7de0107eb71aa064f/GPUtil_fix-1.4.1.2-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"a716d22d0539c6d0e6aea35fad411a896c05e95608e26d36fba73c3e67a6d533","md5":"b60807f831a7c1682ecb2c0b9e52d76a","sha256":"6346639eb5d9c3ddf45213af6fbf7af0dbbd5310a6c24974419c1fc5b19c9774"},"downloads":-1,"filename":"gputil_fix-1.4.1.2.tar.gz","has_sig":false,"md5_digest":"b60807f831a7c1682ecb2c0b9e52d76a","packagetype":"sdist","python_version":"source","requires_python":null,"size":18533,"upload_time":"2024-07-23T12:51:06","upload_time_iso_8601":"2024-07-23T12:51:06.854142Z","url":"https://files.pythonhosted.org/packages/a7/16/d22d0539c6d0e6aea35fad411a896c05e95608e26d36fba73c3e67a6d533/gputil_fix-1.4.1.2.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"1.4.2":{"info":{"author":"Anders Krogh Mortensen","author_email":"anderskroghm@gmail.com","bugtrack_url":null,"classifiers":["Programming Language :: Python :: 3","Programming Language :: Python :: 3.12"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/atarwn/GPUtil-fix","keywords":null,"license":null,"maintainer":"atarwn","maintainer_email":"a@qwa.su","name":"GPUtil-fix","package_url":"https://pypi.org/project/GPUtil-fix/","platform":null,"project_url":"https://pypi.org/project/GPUtil-fix/","project_urls":{"Homepage":"https://github.com/atarwn/GPUtil-fix"},"provides_extra":null,"release_url":"https://pypi.org/project/GPUtil-fix/1.4.2/","requires_dist":null,"requires_python":null,"summary":"Utility to get the GPU status from NVIDA GPUs using nvidia-smi","version":"1.4.2","yanked":true,"yanked_reason":"ImportError: cannot import name 'spawn' from 'shutil'"},"last_serial":24254433,"urls":[{"comment_text":"","digests":{"blake2b_256":"26c36000f5ef95f02093b7d99c60205d7376d5e176aa6f5e8910f5abf584bf19","md5":"b4cf134d255fe38379d2f6acc42b3a3f","sha256":"c066b6e872c7e536c32713d238640e4b7ecc4e744e7b4f62bd1d21e7d2ff154a"},"downloads":-1,"filename":"GPUtil_fix-1.4.2-py3-none-any.whl","has_sig":false,"md5_digest":"b4cf134d255fe38379d2f6acc42b3a3f","packagetype":"bdist_wheel","python_version":"py3","requires_python":null,"size":13925,"upload_time":"2024-07-23T12:33:40","upload_time_iso_8601":"2024-07-23T12:33:40.281612Z","url":"https://files.pythonhosted.org/packages/26/c3/6000f5ef95f02093b7d99c60205d7376d5e176aa6f5e8910f5abf584bf19/GPUtil_fix-1.4.2-py3-none-any.whl","yanked":true,"yanked_reason":"ImportError: cannot import name 'spawn' from 'shutil'"},{"comment_text":"","digests":{"blake2b_256":"7f618c9a72aaf41a2ff7d75994f5a8165d5645414d87ebfe4baa5c85ac15a5c9","md5":"746aca696ca9b49a0c07268c5f69b3ee","sha256":"d41f33ce0de078b553c0cd6a942482492e21c425e9afa5ec9a1934f2865f381c"},"downloads":-1,"filename":"gputil_fix-1.4.2.tar.gz","has_sig":false,"md5_digest":"746aca696ca9b49a0c07268c5f69b3ee","packagetype":"sdist","python_version":"source","requires_python":null,"size":18544,"upload_time":"2024-07-23T12:33:41","upload_time_iso_8601":"2024-07-23T12:33:41.944285Z","url":"https://files.pythonhosted.org/packages/7f/61/8c9a72aaf41a2ff7d75994f5a8165d5645414d87ebfe4baa5c85ac15a5c9/gputil_fix-1.4.2.tar.gz","yanked":true,"yanked_reason":"ImportError: cannot import name 'spawn' from 'shutil'"}],"vulnerabilities":[]}}